chemreg/lists/models.py

```diff
@@ -25,3 +28,50 @@ class IdentifierType(ControlledVocabulary):
     """
 
     pass
+
+
+class ExternalContact(CommonInfo):
+    """External Contact Information
+
+    Attributes:
+        name (str): (Less than 49 characters, unique, required)
+        email (str): (Less than 49 characters, unique, required)
+        phone (str): (Less than 15 characters, unique, required)
+    """
+
+    name = models.CharField(max_length=49, unique=True, blank=False)
+    email = models.CharField(max_length=49, unique=True, blank=False)
+    phone = models.CharField(max_length=15, unique=True, blank=False)
+
+
+class List(ControlledVocabulary):
```

Comment:
Can you double check with Chris that this is supposed to be a controlled vocabulary?  His original model request does not ask for the "Depricated" field.

Reply:
👍 See my comment above